docs/src/main/asciidoc/security-authorize-web-endpoints-reference.adoc

```diff
@@ -782,10 +794,12 @@ quarkus.http.auth.policy.role-policy3.permission-class=org.acme.library.MediaLib
 quarkus.http.auth.permission.roles3.paths=/library/*
 quarkus.http.auth.permission.roles3.policy=role-policy3
 ----
-
 <1> Grants the permission `media-library`, which permits `read`, `write`, and `list` actions.
 Because `MediaLibrary` is the `TvLibrary` class parent, a user with the `admin` role is also permitted to modify `TvLibrary`.
 
+TIP: The `/library/*` path can be tested from a Keycloak provider DEV UI page, because user `alice` created automatically
+by the xref:security-openid-connect-dev-services.adoc[Dev Services for Keycloak] has an `admin` role.
```

Comment:
As the paths are using PUT method shouldn't be better link to section with testing with Swagger UI or GraphQL UI? 
It will be looking like this `xref:security-openid-connect-dev-services.adoc#test-with-swagger-graphql`

Reply:
yes, that's definitely true, but not every application has Swagger UI, while every application with the OIDC extension will have Keycloak provider that links to Swagger UI. I also think that `security-openid-connect-dev-services` contains sections for other testing options (depending if it is web app or service app), so I'd keep it.